modules/apps/data-engine/data-engine-taglib/src/main/resources/META-INF/resources/data_layout_builder/new-js/DataEngineFormBuilder.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import {Context as ModalContext} from '@clayui/modal';
+import {DragLayer} from 'data-engine-taglib';
```

Comment:
🤔 I think that in these cases we can go back to a folder and get it straight from JS. I almost had the feeling that all of this was out of `data-engine-taglib` 😅

Reply:
🤦🏼 🤦🏼 🤦🏼 🤦🏼 🤦🏼 🤦🏼